examples/demo13.py

```diff
@@ -0,0 +1,378 @@
+
+                # Find the overlapping bounds between the large image and the individual postage
+                # stamp.
+                bounds = stamp_bounds & final_image.bounds
+
+                # Copy the image into the right place in the big image.  There is a difference in
+                # normalization between COSMOS images and these that we account for using a
+                # numerical factor.
+                final_image[bounds] += stamp[bounds]/0.03**2
+
+        # Now we're done with the per-galaxy drawing for this image.  The rest will be done for the
+        # entire image at once.
+        logger.info('Postage stamps of all galaxies drawn on a single big image for this filter.')
+        logger.info('Adding the sky level, noise and detector non-idealities.')
+
+        # First we get the amount of zodaical light for a position corresponding to the center of
+        # this SCA.  Since we have supplied an exposure time, the results will be returned to us in
+        # e-/pix.  Then we multiply this by a factor >1 to account for the amount of stray light
+        # that is expected.  Technically one should make a position-dependent sky level, but this
+        # will be a fairly flat function of position, so using a constant is not too bad.
+        sky_level_pix = wfirst.getSkyLevel(filters[filter_name], position=SCA_cent_pos,
+                                           exp_time=wfirst.exptime)
+        sky_level_pix *= (1.0 + wfirst.stray_light_fraction)
+        # Finally we add the expected thermal backgrounds in this band.  These are provided in
+        # e-/pix/s, so we have to multiply by the exposure time.
+        sky_level_pix += wfirst.thermal_backgrounds[filter_name]*wfirst.exptime
+        # Adding sky level to the image.  
+        final_image += sky_level_pix
+
+        # Now that all sources of signal (from astronomical objects and background) have been added,
+        # we can include the expected Poisson noise:
+        final_image.addNoise(poisson_noise)
+
+        # The subsequent steps account for the non-ideality of the detectors.
+
+        # 1) Reciprocity failure:
+        # Reciprocity, in the context of photography, is the inverse relationship between the
+        # incident flux (I) of a source object and the exposure time (t) required to produce a
+        # given response(p) in the detector, i.e., p = I*t. However, in NIR detectors, this
+        # relation does not hold always. The pixel response to a high flux is larger than its
+        # response to a low flux. This flux-dependent non-linearity is known as 'reciprocity
+        # failure', and the approximate amount of reciprocity failure for the WFIRST detectors is
+        # known, so we can include this detector effect in our images.
+
+        # Save the image before applying the transformation to see the difference
+        final_image_orig = final_image.copy()
```

Comment:
Recommend removing this and all the diff images.  I don't think they are all that instructive.  And people who are interested in seeing them will know how to make them themselves.